.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: objective-c
+osx_image: xcode11.2
+install:
+  - npm install
```

Comment:
Is this root npm install needed?

Reply:
Yep, it installs React Native and tools.